src/simularium/VisGeometry/GeometryStore.ts

```diff
@@ -297,25 +297,29 @@ class GeometryStore {
         const objLoader = new OBJLoader();
         this.prepMeshRegistryForNewObj(url);
         return new Promise((resolve, reject) => {
-            objLoader.load(
-                url,
-                (object: Object3D) => {
-                    this.handleObjResponse(url, object);
-                    resolve(object);
-                },
-                (xhr) => {
-                    this.mlogger.info(
-                        url,
-                        " ",
-                        `${(xhr.loaded / xhr.total) * 100}% loaded`
-                    );
-                },
-                (error) => {
-                    // if the request fails, leave agent as a sphere by default
-                    this.mlogger.warn("Failed to load mesh: ", error, url);
-                    return reject(`Failed to load mesh: ${url}`);
-                }
-            );
+            try {
```

Comment:
This try-catch addition isn't related to the three.js update, right? (Fine with me, just wondering)

Reply:
the try catch allows me to handle the error in nodejs not knowing about Request (the fetch API) when we run unit tests that call this code.

Reply:
Oh... maybe if the `error` type caught in the catch block in the test environment is specific enough it'd be good to only handle that type of error? And maybe also add a comment saying the try-catch block was added for the test environment? Otherwise I'm scratching my head why we need error handling in both line 314 and line 320. Not a huge deal though...